payroll-processor-functions/Payroll Processor.postman_collection.json

```diff
@@ -0,0 +1,156 @@
+{
+	"info": {
+		"_postman_id": "cca9cfc4-7e8e-4bfb-9c39-dcdb76258948",
+		"name": "Payroll Processor",
```

Comment:
You can load this up in Postman to be able to more easily send requests to the Functions without the client needing to be hooked up, since we don't have form submission in the client yet or default data populating the Azure Tables.